scripts/repackage-windows-zip.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-2024  The DOSBox Staging Team
+
+# Repackages the Windows ZIP release package generated by the CI workflow
+# under a parent directory.
+#
+# Prerequisites: zip & unzip commands
+
+set -e
+
+if [ $# -ne 1 ]; then
+	echo "Usage: repackage-windows-zip.sh MSYS2_ZIP_NAME"
+	exit 1
+fi
+
+IN_ZIP=$1
+IN_ZIP_PREFIX=dosbox-staging-windows-msys2-x86_64-
+
+if [[ $IN_ZIP != $IN_ZIP_PREFIX* ]]; then
+	echo "Input filename must start with '$IN_ZIP_PREFIX'"
+	exit 1
+fi
+
+function strip {
+	local STRING=${1#$"$2"}
+	echo "${STRING%$"$2"}"
+}
+
+VERSION=$(strip "$IN_ZIP" $IN_ZIP_PREFIX)
+ROOT_DIR=dosbox-staging-$VERSION
+OUT_ZIP=dosbox-staging-windows-$VERSION
+
+RMDIR_CMD="rm -rf $ROOT_DIR"
```

Comment:
I just read about a bug the other day where a script erased an entire drive due to a line like this where the variable (under certain circumstances) could be an empty string.  Probably here that can't happen but `rm -rf $VARIABLE` always gives me a bit of a pause.

Reply:
Yeah, it could be evil, I agree. It also gave me a pause when I wrote it, but the alternative of potentially having leftover files in the target dir from a previous run is not ideal either. Like you said, it's safe as `$ROOT_DIR` always starts with `dosbox-staging-` (as safe as deleting stuff with `rm` can get... ;) )